buildlib/az-helpers.sh

```diff
@@ -214,15 +214,26 @@ check_release_build() {
     build_reason=$1
     build_sourceversion=$2
     title_mask=$3
+    launch=False
 
+    # DRP release scheduled testing
+    if [ "${build_reason}" == "Schedule" ]; then
+        launch=True
 
-    if [ "${build_reason}" == "IndividualCI" ] || \
-       [ "${build_reason}" == "ResourceTrigger" ]
+    elif [ "${build_reason}" == "IndividualCI" ] || \
+         [ "${build_reason}" == "ResourceTrigger" ]
     then
-        launch=True
+        if [[ "$BUILD_DEFINITIONNAME" == *"DRP" ]]
+    then
```

Comment:
wrong indent

Reply:
It was a bit more readable that way😪

Reply:
IMO it's less readable because it does not comply with the if nesting level

Reply:
"then" was definitely wrongly indented, thx